app/controllers/application_controller.rb

```diff
@@ -1,2 +1,4 @@
 class ApplicationController < ActionController::Base
+  include DeviseTokenAuth::Concerns::SetUserByToken
+  protect_from_forgery with: :null_session
```

Comment:
Yes and not. Please add a `TODO` comment here. 
When we will start configuring controller we will transfer this to the ApiController.

Reply:
This line is here to avoid CSRF attack on for rails admin we will need `:exception` option ;)